### DISCUSSION TITLE
Welcome to Autopilot-Management Discussions!

### DISCUSSION THREAD

**Discussion by Jaekty** [Jaekty/Autopilot-Management#1] [Announcements]
If you have any questions or ideas you may write here

**Comment by dpilgrim0702** [comment]
I downloaded the new version 1.1.2 but receiving error "This app can't run on your PC".   I have tried running on Windows 11 23h2 and on Server 2016.

**Reply by Jaekty** [reply]
I tested on a freshly installed Windows 11 23H2, running without issues.
Do you have any security configurations that might block it from running? 
Also worth noting, you must be able to execute commands through PowerShell 5.1.
Can you try running the source code (ps1 script) directly?

**Reply by dpilgrim0702** [reply]
I was able to run it only by using PowerShell ISE x86 running the ps1 script.  Thank you

**Comment by tricky0025** [comment]
Hello, can you please add option to not delete hash? It's needed when resetting pc for new setups. It's nice that it does delete to start fresh in some cases, but the option would be helpful...thanks you for this awesome tool!

**Reply by Jaekty** [reply]
Hello, I will look into the possibility of adding this feature. Thanks for the feedback.

**Reply by Jaekty** [reply]
Option to delete only Intune devices (or only Autopilot) have been added in the latest release published just now.
